index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Cindy Kandie is a well equipped frontend developer who has taken time to polish her craft. This Portfolio is a visual representation of the work she has done and what she can do." />
+    <meta name="keywords" content="frontend developer, web developer" />
+    
+
+    <link rel="stylesheet" href="styles.css">
+    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Poppins:wght@200;400;500;600;700;800;900&display=swap" rel="stylesheet">
+
+    <title>Cindy Kandie's Web Development Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <p>Cindy K</p>
+        </div>
```

Comment:
- [Optional] kindly, use the `<a>` tag for those reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.